src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+                privateProviderCity: {
+                  'ui:title': 'City',
+                },
+                privateProviderPostalCode: {
+                  'ui:title': 'Postal code',
+                  'ui:options': {
+                    widgetClassNames: 'usa-input-medium',
+                  },
+                },
+                privatePrimaryPhoneNumber: {
+                  'ui:title': 'Primary phone number',
+                  'ui:widget': PhoneNumberWidget,
+                  'ui:options': {
+                    widgetClassNames: 'va-input-medium-large',
+                  },
+                  'ui:errorMessages': {
+                    pattern: 'Phone numbers must be 10 digits (dashes allowed)',
+                  },
+                },
+                limitedConsent: {
+                  'ui:title':
+                    'I give consent, or permission, to my doctor to release only records related to [condition].',
+                },
+                'view:privateRecordsChoiceHelp': {
+                  'ui:description': limitedConsentDescription,
+                },
+              },
+            },
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              [formFields.privateMedicalProvider]: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    privateProviderName: {
+                      type: 'string',
+                    },
+                    limitedConsent: {
+                      type: 'boolean',
+                    },
+                    'view:privateRecordsChoiceHelp': {
+                      type: 'object',
+                      properties: {},
+                    },
+                    dateRange: {
+                      $ref: '#/definitions/dateRange',
+                    },
+                    privateProviderCountry: {
+                      type: 'string',
+                    },
+
+                    privateProviderStreetAddress: {
+                      type: 'string',
+                    },
+                    privateProviderCity: {
+                      type: 'string',
+                    },
+                    privateProviderState: {
+                      type: 'string',
+                    },
+                    privateProviderPostalCode: {
+                      type: 'string',
+                    },
+                    privatePrimaryPhoneNumber: {
+                      type: 'string',
+                    },
+                  },
+                  required: [
+                    'privateProviderName',
+                    'privateProviderStreetAddress',
+                  ],
+                },
+              },
+            },
+          },
+        },
+        [formPages.contactInfo]: {
+          path: 'summary-information',
+          title: 'Summary Information',
+          uiSchema: {
+            'ui:title': 'Summary of evidence',
```

Comment:
This will be a summary of all evidence when integrated with 526.

![screen shot 2018-08-29 at 1 44 41 pm](https://user-images.githubusercontent.com/41440372/44805279-c051e400-ab91-11e8-9ae0-c0af01f0c9b3.png)